runtime/testing-basilisk/src/lib.rs

```diff
@@ -304,6 +308,7 @@ impl pallet_transaction_multi_payment::Config for Runtime {
 	type WithdrawFeeForSetCurrency = MultiPaymentCurrencySetFee;
 	type WeightToFee = WeightToFee;
 	type NativeAssetId = ();
```

Comment:
hm.not part of this PR: but why there is a missing native asset id ? i wonder how this could work without it 🤔

Reply:
https://github.com/paritytech/substrate/blob/4037345b8c0491ae5b74fcb6d131b3ff304f0bed/primitives/runtime/src/traits.rs#L297

Reply:
well.that means we just lucky that native is 0 then. 

better to fix it.

Reply:
#495